config/generic-mellow-fly-e3-v2.cfg

```diff
@@ -0,0 +1,244 @@
+# Copyright (C) 2024 Lieven Vanhercke <lieven.vanh@gmail.com>
+
+# This file may be distributed under the terms of the GNU GPLv3 license.
+
+# This file contains common pin mappings for the Mellow Fly-E3-v2.
+# To use this config, the firmware should be compiled for the
+# STM32F407 with a "32KiB bootloader".
+
+# The "make flash" command does not work on the Fly-E3-v2. Instead,
+# after running "make", copy the generated "out/klipper.bin" file to a
+# file named "firmware.bin" or "klipper.bin" on an SD card and then restart the Fly-E3-v2
+# with that SD card.
+
+# See docs/Config_Reference.md for a description of parameters.
+
+[mcu]
+serial: /dev/serial/by-id/usb-Klipper_stm32f407xx_27004A001851323333353137-if00
+
+[stepper_x]
+step_pin: PE5
+dir_pin: PC0
+enable_pin: !PC1
+microsteps: 16
+rotation_distance: 30
+full_steps_per_rotation: 200
+endstop_pin: PE7 #X-STOP
+position_endstop: 0
+position_max: 200
+homing_speed: 50
+second_homing_speed: 10
+homing_retract_dist: 5.0
+homing_positive_dir: false
+step_pulse_duration: 0.000004
+
+[stepper_y]
+step_pin: PE4
+dir_pin: !PC13
+enable_pin: !PC14
+microsteps: 16
+rotation_distance: 30
+full_steps_per_rotation: 200
+endstop_pin: PE8 #Y-STOP
+position_endstop: 0
+position_max: 200
+homing_speed: 50
+second_homing_speed: 10
+homing_retract_dist: 5.0
+homing_positive_dir: false
+step_pulse_duration: 0.000004
+
+[stepper_z]
+step_pin: PE1
+dir_pin: !PB7
+enable_pin: !PE3
+microsteps: 16
+rotation_distance: 30
+full_steps_per_rotation: 200
+endstop_pin: PE9 #Z-STOP
+position_min: 0
+position_endstop: 0
+position_max: 200
+homing_speed: 5
+second_homing_speed: 3
+homing_retract_dist: 5.0
+homing_positive_dir: false
+step_pulse_duration: 0.000004
+
+[extruder]
+step_pin: PE2
+dir_pin: PD5
+enable_pin: !PD6
+microsteps: 16
+rotation_distance: 33.500
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+heater_pin: PC6 #E0
+
+########################################
+# Extruder 100K thermistor configuration
+########################################
+sensor_type: ATC Semitec 104GT-2
+sensor_pin: PC4 #T0 TEMP
+control: pid
+pid_Kp: 22.2
+pid_Ki: 1.08
+pid_Kd: 114
+min_temp: 0
+max_temp: 275
+########################################
+# Extruder MAX31865 PT100 2 wire config
+########################################
+# sensor_type: MAX31865
+# sensor_pin: PD15 #PT-100
+# spi_speed: 4000000
+# spi_software_sclk_pin: PD12
+# spi_software_mosi_pin: PD11
+# spi_software_miso_pin: PD13
+# rtd_nominal_r: 100
+# rtd_reference_r: 430
+# rtd_num_of_wires: 2
+# rtd_use_50Hz_filter: True
+min_temp: 0
+max_temp: 300
+
+#[extruder1]
+#step_pin: PE0
+#dir_pin: PD1
+#enable_pin: !PD3
+#microsteps: 16
+#heater_pin: PC7 #E1
+#sensor_pin: PC5 #T1 TEMP
+
+########################################
+# TMC2209 configuration
+########################################
+
+[tmc2209 stepper_x]
+uart_pin: PC15
+interpolate: False
+run_current: 0.3
+sense_resistor: 0.110
+stealthchop_threshold: 500
```

Comment:
Please do not set a non 0 or 999999 figure for stealthchop_threshold, the option should be on or off. Please see https://github.com/Klipper3d/klipper/blob/master/docs/TMC_Drivers.md#setting-spreadcycle-vs-stealthchop-mode

This goes for all tmc sections

Thanks
James